src/Lykke.Job.TransactionHandler/Projections/OperationHistoryProjection.cs

```diff
@@ -135,7 +134,7 @@ public class OperationHistoryProjection
 
         public async Task Handle(TradeCreatedEvent evt)
         {
-            await _log.WriteInfoAsync(nameof(OperationHistoryProjection), nameof(TradeCreatedEvent), evt.ToJson(), "");
+            _log.WriteInfo(nameof(TradeCreatedEvent), evt.ToJson(), "");
```

Comment:
This is not needed with latest version of Lykke.Cqrs - such logging is done from the library